debezium-connector-postgres/src/main/java/io/debezium/connector/postgresql/PostgresValueConverter.java

```diff
@@ -621,4 +631,18 @@ private boolean isVariableScaleDecimal(final Column column) {
 
         return Optional.empty();
     }
+
+    protected Object convertTimestampToUTC(Column column, Field fieldDefn, Object data) {
+        if (data == null) {
+            return null;
+        }
+        if (!(data instanceof Timestamp)) {
+            return data;
+        }
+        final Timestamp timestamp = (Timestamp) data;
+        final LocalDateTime serverLocalTime = timestamp.toInstant().atZone(ZoneId.systemDefault()).toLocalDateTime();
```

Comment:
It seems this would be a bit simpler:

    serverLocalTime = timestamp.toInstant().toLocalDateTime();

Reply:
There is no `toLocalDateTime()` on `Instant`